aeron-client/src/main/cpp/concurrent/logbuffer/LogBufferDescriptor.h

```diff
@@ -354,7 +354,7 @@ inline static void rotateLog(AtomicBuffer& logMetaDataBuffer, std::int32_t curre
     LogBufferDescriptor::casActiveTermCount(logMetaDataBuffer, currentTermCount, nextTermCount);
 }
 
-inline static void initializeTailWithTermId(AtomicBuffer& logMetaDataBuffer, int partitionIndex, std::int32_t termId)
+inline void initializeTailWithTermId(AtomicBuffer& logMetaDataBuffer, int partitionIndex, std::int32_t termId)
```

Comment:
Why remove the static declaration from methods? Would this not be a breaking API change?

Reply:
It may break the ABI in case someone wants to compile object files with the new header and link with the client library that was compiled with the old headers. With a simple test case, I confirmed that two object files still link if one of them is `inline static` and the other is only `inline`. The advantage is that the resulting binary is smaller as the linker is now free to eliminate repeating `inline` symbols as they are not internally linked.